src/main/java/com/cozyhome/onlineshop/reviewservice/service/ReviewServiceImpl.java

```diff
@@ -48,6 +50,7 @@ public ReviewResponse addNewReview(ReviewRequest reviewRequest, String userId) {
     public List<ReviewResponse> getReviewsForProduct(String productSkuCode) {
         List<Review> reviews = repository.findReviewsByProductSkuCode(productSkuCode);
         if (reviews.isEmpty()) {
+            log.info("[ON getReviewsForProduct]:: Review for product with sku code = {} isn't exist.", productSkuCode);
             return new ArrayList<>();
         }
```

Comment:
we can add filter method to stream in the ReviewBuilder:
"public List<ReviewResponse> buildReviewsResponse(List<Review> reviews) {
        return reviews.stream().filter(review -> review!=null).map(this::buildReviewResponse).toList();
    }"

Reply:
done